src/runtime/nitro/plugins/04-cspSsgHashes.ts

```diff
@@ -43,13 +56,9 @@ export default defineNitroPlugin((nitroApp) => {
               if (!src && scriptText) {
                 // Hash inline scripts with content
                 scriptHashes.add(`'${generateHash(scriptText, hashAlgorithm)}'`)
```

Comment:
@vejja I have a pattern in mind for inline scripts, but I don't know how cheerio works in this case:
Do we need to make sure the contents of the script is non-empty-ish? That is, no whitespace only? 
Because the pattern would change accordingly;
If it's only that it's not empty, this can be used:
```ts
/<script(?![^>]*?src="[\w:.-\\/]+")>(.*?)<\/script>/g
```
If it's that it's not whitespace only, this can be used:
```ts
/<script(?![^>]*?src="[\w:.-\\/]+")>(?!\s*?<\/script>)(.+?)<\/script>/g 
```
The difference is the negative lookahead in the second pattern to make sure it's not only whitespace until the closing tag, which entirely depends on cheerio's functionality in such cases.

Reply:
Hi @GalacticHypernova 
I need to do a full review, but in the meantime I just want to understand your question above.
1- inline scripts don't have the `src` attribute
2- external scripts have the `src` attribute. They can have an `innerHtml` content but it will be ignored
Cheerio parses the `src` attribute and the `innerHtml` node separately. These are two different objects, located in two different places.

Reply:
Alright, let's keep it as is then